.github/workflows/test.yml

```diff
@@ -70,7 +70,7 @@ jobs:
           python-version: "3.10"
 
       - name: Install dependencies
-        run: pip install -r test/requirements.txt
+        run:  pip install wheel && pip install -r test/requirements.txt
```

Comment:
Should this be an `&&` that could fail, thus skipping the `install -r test/requirements.txt` portion of the step?

Can it be broken down to two `run` steps?

Reply:
I can break it down into two steps. `install -r test/requirements.txt` should only be executed if `wheel` is installed successfully. At the moment it won't matter, as `wheel` is not a hard requirement. But it will be once `pip` is v23.1

Reply:
If the first one fails, it will be possible to continue?